sipa/blueprints/generic.py

```diff
@@ -162,7 +165,18 @@ def logout():
                 extra={'tags': {'user': current_user.uid}})
     logout_user()
     flash(gettext("Abmeldung erfolgreich!"), 'success')
-    return redirect(url_for('.index'))
+
+    resp = redirect(url_for('.index'))
+
+    cookie_name = current_app.config.get('REMEMBER_COOKIE_NAME', COOKIE_NAME)
+    if cookie_name in request.cookies:
+        resp.set_cookie(cookie_name, "", 0)
+
+    current_app.login_manager._clear_cookie(resp)
+    resp.set_cookie("session", "")
+    session.clear()
+
+    return resp
```

Comment:
I see what he tried to do, but I don't like it. He's calling a private method. He's using set_cookie instead of delete_cookie. And this code in general seems superfluous and should better done elsewhere, if it all.

No need to tell me to look at the sources. I already checked the source and there is [this](/maxcountryman/flask-login/blob/14e24935baf92f79233cb342dbc93c5ac546439f/flask_login/login_manager.py#L117), [this](/maxcountryman/flask-login/blob/14e24935baf92f79233cb342dbc93c5ac546439f/flask_login/login_manager.py#L402) and [this](/maxcountryman/flask-login/blob/14e24935baf92f79233cb342dbc93c5ac546439f/flask_login/login_manager.py#L441).

If the referenced logic does not delete the `REMEMBER_COOKIE_NAME` then we have bug somewhere.